### ISSUE TITLE
Problem after updating to Mac OS 12.3 (21E230)

### ISSUE THREAD

**Issue by kotlinski** [olivierverdier/zsh-git-prompt#153]
Stopped working as expected after 12.3 (21E230)

Before I got the number of diffs (up and down), now I just get the ⚡ icon.

**Comment by kotlinski** [comment]
~/dev/zsh-git-prompt(master⚡) » 

Before I used to have the number of changed files.

**Comment by buko106** [comment]
cause of this problem may be that `/usr/bin/python` is deleted (`/usr/bin/python3` still exists)

c.f. https://zenn.dev/sprout2000/articles/df9d59336a51d5 (sorry for Japanese blog only)

**Comment by kotlinski** [comment]
Yes, I figured that out after a few hours of debugging. I ended up adding an alias in .zshrc
`alias python=/usr/local/bin/python3.9`

**Comment by patrick-mcdougle** [comment]
Depending on circumstances, you might also need to install the xcode command line tools as described here: https://apple.stackexchange.com/questions/254380/why-am-i-getting-an-invalid-active-developer-path-when-attempting-to-use-git-a

**Comment by hydrospanner** [comment]
> Yes, I figured that out after a few hours of debugging. I ended up adding an alias in .zshrc `alias python=/usr/local/bin/python3.9`

This will cause issues for anything attempting to run python2 code. Since this repo isn't being maintained use https://github.com/zsh-git-prompt/zsh-git-prompt instead. It works on systems with only python3.
